testsuite/tests/typing-misc/polyvars.ml

```diff
@@ -9,12 +9,12 @@ Error: This pattern matches values of type [? `A | `B ]
 |}];;
 let f x = ignore (match x with #ab -> 1); ignore (x : [`A]);;
 [%%expect{|
-Line _, characters 31-34:
+Line _, characters 32-34:
```

Comment:
One could argue that the pattern here is `#ab`,  not `ab`, so the old location might be better.